R/coord-cartesian-.r

```diff
@@ -153,10 +214,41 @@ view_scales_from_scale <- function(scale, coord_limits = NULL, expand = TRUE) {
   view_scales
 }
 
-draw_view_scale_axis <- function(view_scale, axis_position, theme) {
-  if(is.null(view_scale) || view_scale$is_empty()) {
+panel_guide_label <- function(guides, position, default_label) {
```

Comment:
Shouldn't all these helpers below be placed in `coord-.r` they seem global to all coords?